frontend/src/components/inputs/AutoCompleteCourseSelect.js

```diff
@@ -1,23 +1,16 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Autosuggest from 'react-autosuggest';
 import { Link } from 'react-router-dom';
 import styles from './_autocomplete-course-select.scss';
 import classNames from 'classnames/bind';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import apiConfig from 'base/apiConfig';
 
 let cx = classNames.bind(styles);
 
-var coursesList = [
-];
-
-const getSuggestions = value => {
-  const inputValue = value.trim().toLowerCase();
-  const inputLength = inputValue.length;
-  return inputLength === coursesList ? [] : coursesList.filter(course => ((course.courseName.toLowerCase().slice(0, inputLength) === inputValue) || (course.courseNumber.toLowerCase().slice(0, inputLength) === inputValue))).toArray();
-};
+const WAIT_INTERVAL = 1000;
```

Comment:
@grozdanowski How helpful would being able to pull configuration data from the server? I see this as an example where we might want to tune what the wait interval is without having to deploy updated code

Reply:
@johnbaldwin we can pass this as an env variable.  I think it's outside the scope for this PR, but we could do quite a lot of settings of the app that way, making it more flexible.